### PR TITLE
[Balance] Remove Species Locks from Aura Wheel and Poison Puppeteer

### PR DESCRIPTION
**Pull request by innerthunder**
## What are the changes the user will see?
- Aura Wheel is now functional for species other than Morpeko (and fusions with Morpeko). It defaults to Electric type when used by Pokemon other than Morpeko
- Poison Puppeteer is now functional for species other than Pecharunt

## Why am I making these changes?
These species locks are unnecessarily restrictive with the existence of fusions, egg moves, and passive abilities.

## What are the changes from a developer perspective?
- `data/move`: Removed Aura Wheel's condition requiring the user to be Morpeko (or a Morpeko fusion)
- `data/ability`: Made Poison Puppeteer's effect attribute unconditional to the source's species.
- Localization changes:
  - Replaced reference to Pecharunt in Poison Puppeteer's description with "this Pokémon"
  - Updated Aura Wheel's description to reflect the changes to species-related conditions

## How to test the changes?
Using `src/overrides.ts`:
- Give a Pokemon other than Pecharunt Poison Puppeteer and Toxic. You should see that Toxic also confuses the target.
- Give a Pokemon other than Morpeko Aura Wheel. You should see that the move doesn't fail and defaults to Electric type.

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I considered writing automated tests for the issue?
- [x] If I have text, did I make it translatable and add a key in the English locale file(s)?
- [ ] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by damocleas — DISMISSED** [review]

**Review by EnochG1 — DISMISSED** [review]
Putting '// TODO' in string instead of comments will make display wrong.

**Comment by innerthunder** [comment]
> Putting '// TODO' in string instead of comments will make display wrong.

json doesn't support comments, so moving these outside of the string would cause the code not to compile. These markers will be removed before this is taken out of draft, anyway

**Comment by EnochG1** [comment]
@innerthunder 
Assuming this work is merged without the translator modifying the description, 
I think it would be more appropriate to replace it with the changed English description.
(Wrong translation is worse than right English desc.)

File: src/locales/ko/ability.json [hunk 1]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "독조종",
-    "description": "복숭악동의 기술에 의해 독 상태가 된 상대는 혼란 상태도 되어 버린다."
+    "description": "복숭악동의 기술에 의해 독 상태가 된 상대는 혼란 상태도 되어 버린다. //TODO"
```
Comment (EnochG1) [inline src/locales/ko/ability.json:1240]:
```suggestion
    "description": "이 기술에 의해 독 상태가 된 상대는 혼란 상태도 되어 버린다."
```

File: src/locales/ko/move.json [hunk 2]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "오라휠",
-    "effect": "볼주머니에 저장해둔 에너지로 공격하고 자신의 스피드를 올린다. 모르페코의 모습에 따라 타입이 바뀐다."
+    "effect": "볼주머니에 저장해둔 에너지로 공격하고 자신의 스피드를 올린다. 모르페코의 모습에 따라 타입이 바뀐다. //TODO"
```
Comment (EnochG1) [inline src/locales/ko/move.json:3132]:
```suggestion
    "effect": "볼주머니에 저장해둔 에너지로 공격하고 자신의 스피드를 올린다. 모르페코가 사용할 경우 모습에 따라 타입이 바뀐다."
```

**Review by EnochG1 — COMMENTED** [review]
Korean TL

File: src/locales/pt_BR/ability.json [hunk 3]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "Poison Puppeteer",
-    "description": "Pokémon envenenados pelos movimentos de Pecharunt também ficarão confusos."
+    "description": "Pokémon envenenados pelos movimentos de Pecharunt também ficarão confusos. //TODO"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/ability.json:1240]:
```suggestion
    "description": "Pokémon envenenados pelos movimentos deste Pokémon também ficarão confusos."
```

File: src/locales/pt_BR/move.json [hunk 4]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "Aura Wheel",
-    "effect": "Morpeko ataca e aumenta sua Velocidade com a energia armazenada em suas bochechas. O tipo deste movimento muda dependendo da forma do usuário."
+    "effect": "Morpeko ataca e aumenta sua Velocidade com a energia armazenada em suas bochechas. O tipo deste movimento muda dependendo da forma do usuário. //TODO"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/move.json:3132]:
```suggestion
    "effect": "O usuário ataca e aumenta sua Velocidade com a energia armazenada em suas bochechas. Se usado por Morpeko, o tipo deste movimento muda dependendo da forma do usuário."
```

**Review by sodaMelon — APPROVED** [review]

File: src/locales/de/move.json [hunk 5]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "Aura-Rad",
-    "effect": "Mithilfe der in den Backentaschen gespeicherten Energie greift der Anwender an und erhöht seine Initiative. Der Typ der Attacke hängt von Morpekos Form ab."
+    "effect": "Mithilfe der in den Backentaschen gespeicherten Energie greift der Anwender an und erhöht seine Initiative. Der Typ der Attacke hängt von der Form des Pokémon ab."
```
Comment (CodeTappert) [inline src/locales/de/move.json:3132]:
```suggestion
    "effect": "Mithilfe der in den Backentaschen gespeicherten Energie greift der Anwender an und erhöht seine Initiative. Wenn dies von Morpeko verwendet wird hängt der Typ der Attacke von dessen Form ab."
```

File: src/locales/it/ability.json [hunk 6]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "\tMalia Tossica",
-    "description": "I Pokémon avvelenati dalle mosse di Pecharunt entreranno anche in stato di confusione."
+    "description": "I Pokémon avvelenati dalle mosse di Pecharunt entreranno anche in stato di confusione. //TODO"
```
Comment (NicusPulcis) [inline src/locales/it/ability.json:1240]:
```suggestion
    "description": "I Pokémon avvelenati dalle mosse di questo Pokémon entreranno anche in stato di confusione."
```

File: src/locales/it/move.json [hunk 7]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "Ruota d'Aura",
-    "effect": "Il Pokémon emette l'energia accumulata nelle guance per attaccare e aumentare la Velocità. Il tipo della mossa cambia in base alla forma assunta da Morpeko."
+    "effect": "Il Pokémon emette l'energia accumulata nelle guance per attaccare e aumentare la Velocità. Il tipo della mossa cambia in base alla forma assunta da Morpeko. //TODO"
```
Comment (NicusPulcis) [inline src/locales/it/move.json:3132]:
```suggestion
    "effect": "Il Pokémon emette l'energia accumulata nelle guance per attaccare e aumentare la Velocità. Se usata da Morpeko, il tipo della mossa cambia in base alla forma assunta."
```

**Review by NicusPulcis — APPROVED** [review]
italian localized

File: src/locales/ja/ability.json [hunk 8]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "どくくぐつ",
-    "description": "モモワロウの　技によって　どく状態に　なった　相手は　こんらん状態にも　なってしまう。"
+    "description": "モモワロウの　技によって　どく状態に　なった　相手は　こんらん状態にも　なってしまう。 //TODO"
```
Comment (Chapybara-jp) [inline src/locales/ja/ability.json:1240]:
```suggestion
    "description": "このポケモンの　技によって　どく状態に　なった　相手は　こんらん状態にも　なってしまう。"
```

File: src/locales/ja/move.json [hunk 9]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "オーラぐるま",
-    "effect": "ほほぶくろに　溜めた　エネルギーで　攻撃し　自分の　素早さを　あげる。　モルペコの　姿で　タイプが　変わる。"
+    "effect": "ほほぶくろに　溜めた　エネルギーで　攻撃し　自分の　素早さを　あげる。　モルペコの　姿で　タイプが　変わる。//TODO"
```
Comment (Chapybara-jp) [inline src/locales/ja/move.json:3132]:
```suggestion
    "effect": "ほほぶくろに　溜めた　エネルギーで　攻撃し　自分の　素早さを　あげる。　モルペコが　この技を　使う場合　姿で　技の　タイプが　変わる。"
```

File: src/locales/es/ability.json [hunk 10]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "Títere Tóxico",
-    "description": "Los rivales que Pecharunt envenene con sus movimientos también sufrirán confusión."
+    "description": "Los rivales que Pecharunt envenene con sus movimientos también sufrirán confusión. //TODO"
```
Comment (DanStevensonx) [inline src/locales/es/ability.json:1240]:
```suggestion
    "description": "Los rivales que el usuario envenene con sus movimientos también sufrirán confusión."
```

File: src/locales/es/move.json [hunk 11]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "Rueda Aural",
-    "effect": "La energía que acumula en las mejillas le sirve para atacar y aumentar su Velocidad. Este movimiento cambia de tipo según la forma que adopte Morpeko."
+    "effect": "La energía que acumula en las mejillas le sirve para atacar y aumentar su Velocidad. Este movimiento cambia de tipo según la forma que adopte Morpeko. //TODO"
```
Comment (DanStevensonx) [inline src/locales/es/move.json:3132]:
```suggestion
    "effect": "La energía que acumula en las mejillas le sirve para atacar y aumentar su Velocidad. Si es utilizado por Morpeko, este movimiento cambia de tipo según la forma que adopte."
```

**Review by DanStevensonx — COMMENTED** [review]
Here is a haiku
The changes were made, 
It was not hard to do it,
But it just got done.

File: src/locales/zh_CN/ability.json [hunk 12]
```diff
@@ -1237,6 +1237,6 @@
   },
   "poisonPuppeteer": {
     "name": "毒傀儡",
-    "description": "因桃歹郎的招式而陷入中毒状态的\n对手同时也会陷入混乱状态。"
+    "description": "因桃歹郎的招式而陷入中毒状态的\n对手同时也会陷入混乱状态。//TODO"
```
Comment (btsrkex) [inline src/locales/zh_CN/ability.json:1240]:
```suggestion
    "description": "因此宝可梦的招式而陷入中毒状态的对手\n同时也会陷入混乱状态。"
```

File: src/locales/zh_CN/move.json [hunk 13]
```diff
@@ -3129,7 +3129,7 @@
   },
   "auraWheel": {
     "name": "气场轮",
-    "effect": "用储存在颊囊里的能量进行攻击，\n并提高自己的速度。其属性会随着\n莫鲁贝可的样子而改变"
+    "effect": "用储存在颊囊里的能量进行攻击，\n并提高自己的速度。其属性会随着\n莫鲁贝可的样子而改变 //TODO"
```
Comment (btsrkex) [inline src/locales/zh_CN/move.json:3132]:
```suggestion
    "effect": "用储存在颊囊里的能量进行攻击，\n并提高自己的速度。如果由莫鲁贝可使用，\n其属性会随着它的样子而改变"
```

**Review by btsrkex — COMMENTED** [review]
update zh_cn and zh_tw

**Review by Adri1 — APPROVED** [review]

**Review by damocleas — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]

**Review by ben-lear — APPROVED** [review]

**Pull request merged**